packages/lit-html/src/lit-html.ts

```diff
@@ -1230,8 +1234,31 @@ class ChildPart {
   }
 }
 
+/**
+ * A top-level `ChildPart` returned from render that manages the connected state
+ * of `AsyncDirective`s created throughout the tree below it.
+ */
+export type {RootChildPart};
+class RootChildPart extends ChildPart {
+  private __isConnected = true;
+
+  get _$isConnected() {
```

Comment:
I wonder if you could save a couple of bytes by making `_$isConnected` the property? You'd need defineProperty to not error on the base class not having a setter, so maybe not.

Reply:
Considered that, but defineProperty is also slow, in addition to bytes.